cmake/modules/FindSmctemp.cmake

```diff
@@ -0,0 +1,33 @@
+#.rst:
+# FindSmctemp
+# -------
+# Finds the smctemp library
+#
+# This will define the following imported targets::
+#
+#   SMCTEMP::SMCTEMP   - The smctemp library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_SMCTEMP libsmctemp QUIET)
+endif()
```

Comment:
This actually isnt required at all. The lib doesnt install a pkgconfig file (.pc extension), so nothing will ever be found.